flow/Makefile

```diff
@@ -585,11 +585,11 @@ klayout_guides: $(RESULTS_DIR)/5_route.def $(OBJECTS_DIR)/klayout.lyt
 # |  _|  | || |\  || | ___) |  _  || || |\  | |_| |
 # |_|   |___|_| \_|___|____/|_| |_|___|_| \_|\____|
 #
-6_FINAL_FILE = $(RESULTS_DIR)/6_final.$(STREAM_SYSTEM_EXT)
+GDS_FINAL_FILE = $(RESULTS_DIR)/6_final.$(STREAM_SYSTEM_EXT)
```

Comment:
Just curious, what's the motivation for the name change?

I remember that someone changed all `GDS` to `GDSOAS` because of adding an option for OAS files. Maybe the naming needs to be more standardized.

Reply:
Envars starting with numbers are disallowed